shapepipe/pipeline/file_io.py

```diff
@@ -1841,3 +1842,59 @@ def comment(self, comment):
         @data.setter
         def data(self, data):
             self._data = data
+
+
+def get_unit_from_fits_header(header, key):
+    """Get Unit From Fits Header.
+
+    Return coordinate unit corresponding to column with name ``key``
+
+    Parameters
+    ----------
+    header : FITS header
+        header information
+    key : str
+        column name
+
+    Raises
+    ------
+    IndexError
+        if key not in header
+
+    Returns
+    -------
+    class Unit
```

Comment:
I think this should be `astropy.units.Unit`

Reply:
fixed